### DISCUSSION TITLE
Changing permalink structure messes up collapsible sidebar

### DISCUSSION THREAD

**Discussion by pensivedog** [thuliteio/doks#839] [Q&A]
In `config.toml` under `[permalinks]`, setting `docs = "/:sections[1:]/:title/"` successfully changes the url structure, but then the collapsible sidebar sections automatically collapse on page load, whereas they are expanded by default when the permalink isn't modified.

Can anyone else reproduce and confirm this is a bug?

**Comment by genebean** [comment]
I’ve been wondering about this too. I updated my permalink to `docs = "/info/:sections[1:]/:title"` and am also seeing the left side nave collapse on every page load.

**Reply by h-enk** [reply]
Can't confirm. Do you also see this behavior _after_ https://github.com/h-enk/doks/issues/885#issuecomment-1236587942

Don't know if this has an effect, but make sure to have in `./config/_default/params.toml`:

```toml
scrollSpy = false # experimental; needs Bootstrap >= 5.2.0-beta1
```

**Comment by sol2070** [comment]
I can confirm. After removing "/docs/" from the permalinks [doing this](https://github.com/h-enk/doks/discussions/343#discussioncomment-2126343), items in the sidebar always collapses on page load.

**Comment by rachfop** [comment]
I got a fix working. 
In `layouts/partials/sidebar/auto-collapsible-menu.html` I replaced all instances of:

```
{{ $active := in $currentPage.RelPermalink .RelPermalink -}}
```
  
With:
 
```
{{ $adjustedCurrentPagePermalink := printf "/docs%s" $currentPage.RelPermalink }}
{{ $active := in $adjustedCurrentPagePermalink .RelPermalink -}}
```

I believe since the `/docs/` wasn't being used, the pages weren't being caught as active.

**Reply by h-enk** [reply]
Thanks for sharing! I will look into this — I've created an issue: https://github.com/h-enk/doks/issues/1080
